packages/components/src/styleConstants.ts

```diff
@@ -0,0 +1,31 @@
+export const colors = {
+  primary: {
+    BLACK: "#000000",
+    CIVIL_GRAY_1: "#5A5653",
+    CIVIL_GRAY_2: "#7D7373",
+    CIVIL_BLUE_1: "#2B56FF",
+  },
+  accent: {
+    CIVIL_YELLOW: "#FF120", // Civil-wide accent color
```

Comment:
I would enjoy more descriptive names telling what scenarios are they used in

Reply:
I copied these from the styled guide. feel free to color (heh) this in a bit more @juliahimmel !